client/my-sites/term-tree-selector/index.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import TermTreeSelectorTerms from './terms';
```

Comment:
I think we still need this setup to manage `search` coming in to `./terms` as part of the `query` prop.